api/target/test-classes/org/openmrs/module/providermanagement/include/providerManagement-dataset.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I don't think this file was supposed to be / should be committed?

Reply:
In order to accurately test the method, more XML test data needs to be added which is why I committed this file.  Is there another way we should go about adding more data?

Reply:
Adding more XML test data is fine and it looks like you did so (see the diff above) but it looks like also committed an additional copy of the test file that is generated during the build by the surefire plugin, this copy should not be committed... note that this is in the "target" directory which is for files generated by maven and should never be committed.